packages/components/src/components/filter-dropdown/filter-dropdown.tsx

```diff
@@ -30,6 +38,17 @@ const FilterDropdown = ({
                 suffix_icon='IcFilter'
                 should_show_empty_option={false}
                 onChange={onChange}
+                className={undefined}
```

Comment:
Just for my understanding why you added these properties like this?

Reply:
In JSX code this props are not using.But when i migrated to tsx it is throwing error that items are missing.So I added undefined.Can I use any other values?

Reply:
oh got it, I'm working on dropdown right now, after I finished migrating this component to ts these props will fix, for now just left a todo there to make sure that we will change it later

Reply:
okay sure

Reply:
It is SelectNative component

Reply:
@yaswanth-deriv if your card is blocked then instead of having props like this, I think it would be better to move your card to blocked until `<SelectNative/>` ts migration is done.